Source/DafnyLanguageServer.Test/DafnyLanguageServerTestBase.cs

```diff
@@ -15,11 +15,17 @@
 using System.IO;
 using System.IO.Pipelines;
 using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using MediatR;
+using Microsoft.Dafny.LanguageServer.Language;
+using OmniSharp.Extensions.LanguageServer.Client;
 
 namespace Microsoft.Dafny.LanguageServer.IntegrationTest {
   public class DafnyLanguageServerTestBase : LanguageServerTestBase {
-    protected const string SlowToVerify = @"
-lemma {:timeLimit 10} SquareRoot2NotRational(p: nat, q: nat)
+    protected readonly string SlowToVerify = @"
+lemma {:timeLimit 3} SquareRoot2NotRational(p: nat, q: nat)
```

Comment:
Why `const` to `readonly`?

Reply:
`const` was not possible because of the `TrimStart()` call